chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/TestingLabManagerImpl.java

```diff
@@ -75,8 +61,8 @@ public TestingLabDTO create(final TestingLabDTO atl)
         // Create the atl itself
         TestingLabDTO result = testingLabDAO.create(atl);
 
-        // Grant the admin user administrative permission to the ATL
-        addPermission(result, User.ADMIN_USER_ID, BasePermission.ADMINISTRATION);
+        // Grant the user administrative permission to the ATL
+        userPermissionsManager.addAtlPermission(result, Util.getCurrentUser().getId());
```

Comment:
With the new impersonation thing in place, this should probably use `Util.getAuditId()` because as it stands it'd grand the permission to the Impersonated user, not the one doing the impersonation. When ROLEs change in an impersonation state, the impersonating user is the one the ROLEs apply to, so I think the authority on ATL (& ACB if it's not right) should be set up the same way